docs/advanced_usage.rst

```diff
@@ -794,22 +794,55 @@ There are three mechanisms for temporary storage.
 
 To modify which storage mechanism is used, please refer to the setting :ref:`import_export_tmp_storage_class`.
 
-Temporary resources are removed when data is successfully imported after the confirmation step.
-
 Your choice of temporary storage will be influenced by the following factors:
 
 * Sensitivity of the data being imported.
 * Volume and frequency of uploads.
 * File upload size.
 * Use of containers or load-balanced servers.
 
+Temporary resources are removed when data is successfully imported after the confirmation step.
+
 .. warning::
 
     If users do not complete the confirmation step of the workflow,
     or if there are errors during import, then temporary resources may not be deleted.
     This will need to be understood and managed in production settings.
     For example, using a cache expiration policy or cron job to clear stale resources.
 
+Customizable storage
+^^^^^^^^^^^^^^^^^^^^^
+
+If using :class:`~import_export.tmp_storages.MediaStorage` as a storage module, then you can define which storage
+backend implementation is used to handle create / read / delete operations on the persisted data.
+
+If using Django 4.2 or greater, use the `STORAGES <https://docs.djangoproject.com/en/stable/ref/settings/#std-setting-STORAGES>`_
+setting to define the backend, otherwise use :ref:`import_export_default_file_storage`.
+
+You can either supply a path to your own custom storage backend, or use pre-existing backends such as
+`django-storages <https://django-storages.readthedocs.io/>`_.
+
+If no custom storage implementation is supplied, then the Django default handler is used.
+
+For example, if using django-storages, you can configure s3 as a temporary storage location with the following::
```

Comment:
Great docs! This is a really cool concept, seems potentially very valuable for super large files AND if you use the S3 backend on a versioned bucket could give you some automated auditing too :)